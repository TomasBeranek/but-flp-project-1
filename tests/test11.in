U,V,W,X
i,j,k,l
X
X->#
X->iV
X->iU
U->ij
V->kl
X->iiW
W->ijkl
