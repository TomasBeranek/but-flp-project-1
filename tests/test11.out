0,1,2,3,4,5,6,7,8,9,10,11,12
ijkl
12
4,6,8,12
0,i,11
1,j,2
2,k,3
3,l,4
5,l,6
7,j,8
9,i,7
10,k,5
11,i,1
12,i,0
12,i,9
12,i,10
